test/e2e/test_e2e.py

```diff
@@ -0,0 +1,727 @@
+#!/usr/bin/python3
+
+import pytest
+import os
+import socket
+import hashlib
+import subprocess
+import yaml
+import time
+import string
+import re
+import sys
+
+import h5py
+import ismrmrd
+import numpy
+import glob
+import json
+
+import urllib.error
+import urllib.request
+
+from pathlib import Path
+
+
+# Importing h5py on windows will mess with your environment. When we pass the messed up environment to gadgetron
```

Comment:
There are some flippant things like these that have been copied over. I don't think that should be the case, and we don't build for Windows, so it should probably be removed. Which would clean things up.